cmd/queue/main.go

```diff
@@ -99,6 +106,17 @@ func initEnv() {
 	servingService = os.Getenv("SERVING_SERVICE") // KService is optional
 	userTargetPort = util.MustParseIntEnvOrFatal("USER_PORT", logger)
 	userTargetAddress = fmt.Sprintf("127.0.0.1:%d", userTargetPort)
+	userContainerName = util.GetRequiredEnvOrFatal("USER_CONTAINER_NAME", logger)
+
+	enableVarLogCollection, _ = strconv.ParseBool(os.Getenv("ENABLE_VAR_LOG_COLLECTION")) // Optional, default is false
```

Comment:
Given the simplicity of this trick, what are the trade-offs that would lead us to not simply enable this by default?

Reply:
* Cost: why pay for logs you don't need by default?
* Privacy: I am not sure collecting everything in /var/log by default is safe, who knows what is logged there
* Backwards compatibility

Reply:
Can't the log collection and so forth be controlled by either the fluentd config or by deleting the daemonset?

Reply:
Yes, one can modify the fluentd config, but:
1) It is global, there was a discussion about making varlog collection a flag per revision instead of globally. This makes the implementation of such feature easy.
2) It is not always possible (GKE default fluentd config cannot be modified). More generally, it is not necessarily the same people operating Knative and fluentd (the infra).